test/internal/modules/charge-information/lib/helpers.js

```diff
@@ -104,6 +104,25 @@ experiment('internal/modules/charge-information/lib/helpers', () => {
     });
   });
 
+  experiment('.getCurrentBillingAccountAddress', () => {
+    const address1 = { id: 'address1', dateRange: { endDate: null } };
+    const address2 = { id: 'address2', dateRange: { endDate: new Date() } };
```

Comment:
Not that I think it affects the test outcome, but I think the `endDate` here would be a string in the form YYYY-MM-DD